incubator/hnc/hack/test-issue-716-full.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
I'm not sure what this test is doing? This PR isn't about deleting orphans? Or do we not have other tests for this?